src/media/js/previews.js

```diff
@@ -178,8 +178,15 @@ define('previews',
 
         $tray.css('width', z.win.width() + 'px');
         var offset = $tray.offset();
-        desktopMarginLeft += -1 * offset.left;
-        $tray.css('margin-left', desktopMarginLeft + 'px');
+
+        // LTR or RTL?
+        if ($('body.html-rtl').length) {
+            desktopSideMargin -= -1 * offset.left;
```

Comment:
You can do `-offset.left` instead of `-1 * offset.left`. And isn't this the same as `desktopSideMargin += offset.left`?